gateway/src/main/java/ru/neoflex/gateway/service/validation/RestClientValidationService.java

```diff
@@ -0,0 +1,30 @@
+package ru.neoflex.gateway.service.validation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+
+import org.springframework.stereotype.Service;
+import ru.neoflex.gateway.exception.HttpStatusNotOkException;
+
+
+@Service
+public class RestClientValidationService {
+    private final String loanOfferMessage;
+    private static final Logger logger = LoggerFactory.getLogger(RestClientValidationService.class);
+    public RestClientValidationService(
+            @Value("${app.exception.http.status.is-not-ok}") String loanOfferMessage) {
+
+        this.loanOfferMessage = loanOfferMessage;
+    }
+
+
+    public boolean validateLoanOfferResult(HttpStatus httpStatus) {
```

Comment:
тут нам boolean не нужен, либо мы упадём с ошибой, либо ничего не произойдёт. Это и будет нашей валидацией
```suggestion
    public void validateLoanOfferResult(HttpStatus httpStatus) {
```